src/environments/environment.interface.ts

```diff
@@ -1,7 +1,8 @@
 export abstract class IEnvironment {
   abstract readonly production: boolean; // build mode
   abstract readonly siteIds: string[]; // Google analytics site id's
-  abstract readonly host: string; // api proxy host
```

Comment:
removing the host seems like the right thing to do. can i ask what was your thinking behind this?

Reply:
there's no host. was used for paths e.g. api/

Reply:
fair enough. much cleaner this way